ci/open-pullrequest.ps1

```diff
@@ -0,0 +1,29 @@
+param(
```

Comment:
[Suggestion] Is that open-pr script localization specific? Can we move it also to the localization-updates dir?

Reply:
Actually, all loc-related scripts including the folder structure were copied from already existing repos, but sure - I also fix this in google-play repo then.

Reply:
Done.